### PR TITLE
passwordsafe: update to 1.16.0

### PR DESCRIPTION
**Pull request by yopito**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture : **x86_64-musl**
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - [ ] aarch64-musl
  - [ ] armv7l
  - [ ] armv6l-musl
  - [x] aarch64 (glibc)

### CODE REVIEW

File: srcpkgs/passwordsafe/template [hunk 1]
```diff
@@ -23,8 +24,10 @@ else
 fi
 
 pre_configure() {
-	# ykpers-1 includes are hardcoded, add cross base
-	vsed -e "s|-I\(/usr/include/ykpers-1\)|-I${XBPS_CROSS_BASE}\1|" -i CMakeLists.txt
+	if [ ${CROSS_BUILD} ]; then
```
Comment (classabbyamp) [inline srcpkgs/passwordsafe/template:27]:
```suggestion
	if [ -n "${CROSS_BUILD}" ]; then
```

Comment (yopito) [reply srcpkgs/passwordsafe/template:27]: yes but if XBPS_CROSS_BASE is empty, `vsed` errors because nothing changed

**Comment by yopito** [comment]
forced push to remove redundant stuff on `configure_args`

File: srcpkgs/passwordsafe/template (hunk 1 above)
Comment (classabbyamp) [reply srcpkgs/passwordsafe/template:26]: ok then i would suggest
```suggestion
	if [ -n "${CROSS_BUILD}" ]; then
```

**Pull request merged**